server/remote/gitlab/gitlab.go

```diff
@@ -17,33 +17,51 @@ package gitlab
 import (
 	"context"
 	"crypto/tls"
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net"
 	"net/http"
 	"net/url"
-	"strconv"
 	"strings"
 
 	"github.com/woodpecker-ci/woodpecker/server"
 	"github.com/woodpecker-ci/woodpecker/server/model"
 	"github.com/woodpecker-ci/woodpecker/server/remote"
 	"github.com/woodpecker-ci/woodpecker/shared/oauth2"
 
-	"github.com/woodpecker-ci/woodpecker/server/remote/gitlab/client"
+	"github.com/xanzy/go-gitlab"
 )
 
-const DefaultScope = "api"
+const (
+	defaultScope  = "api"
+	perPage       = 100
+	statusContext = "ci/drone"
```

Comment:
Can we change this to `ci/woodpecker` or is it matched by Gitlab to this `SetDroneCIService`?

Reply:
Yes gitlab does not have a Service for woodpecker, we could use generic webhooks instead

Reply:
If desired, I would do a followup pull

Reply:
Okay, we need to test if we can just change it in a follow-up or if we need to add a Woodpecker-service to gitlab / use generic webhooks.